lib/quill_delta.dart

```diff
@@ -463,6 +486,59 @@ class Delta {
     return result;
   }
 
+  /// invert this delta base on baseDelta
```

Comment:
Could you update this doc with more details. I think something like this would be good:

```dart
/// Inverts this delta against [base].
///
/// Returns new delta which negates effect of this delta when applied to [base].
/// This is an equivalent of "undo" operation on deltas.
Delta invert(Delta base) {
  // ...
}
```